.github/workflows/build-app-release.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         run: node scripts/build/android/build.js
 
       - name: Build for Android
-        run: eas build --profile production --platform android --non-interactive --auto-submit --no-wait
+        run: eas build --profile production-android --platform android --non-interactive --auto-submit --no-wait
```

Comment:
@alexrisch I'm not sure about this. I think I introduced it this morning during a rebase/merge conflict.

Reply:
updated to we don't have this diff anymore